interfaces/builtin/mount_control_test.go

```diff
@@ -221,9 +221,13 @@ func (s *MountControlInterfaceSuite) TestSanitizePlugUnhappy(c *C) {
 			`mount-control "where" setting cannot be used: missing closing bracket ']'.*`,
 		},
 		{
-			"mount:\n  - what: /\n    where: /media/*\n    options: [sync,invalid]",
+			"mount:\n  - what: /\n    where: /media/*\n    options: [nofail,gid=2000,invalid]",
```

Comment:
no super clear to me why this was changed instead of adding one more test?

Reply:
I reverted my changes to existing tests and added several new ones. Thanks for the suggestion!